src/api.jl

```diff
@@ -1,55 +1,298 @@
+export InitialGuess, 
+    PeriodicOrbit, 
+    PeriodicOrbitFinder,
+    isdiscretetime,
+    complete_orbit,
+    podistance,
+    minimal_period,
+    uniquepos,
+    isstable,
+    poequal,
+    periodic_orbit,
+    periodic_orbits
+
+import DynamicalSystemsBase
+using LinearAlgebra: norm, eigvals
+
+"""
+A structure that contains an initial guess for a periodic orbit detection algorithms.
+
+    * `u0` - guess of a point in the periodic orbit
+    * `T` - guess of period of the orbit
```

Comment:
In the future, when you make bullet point lists, don't increment them by 4 spaces. Just start immediatelly with the `*`